card/lib/card/loader.rb

```diff
@@ -1,6 +1,16 @@
 # -*- encoding : utf-8 -*-
 
 class Card
+  class << self
```

Comment:
You think this belongs in card.rb?  Another place?  This didn't seem to outlandish and ensure it is there for all module loading.